modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/QueryChecker.java

```diff
@@ -89,6 +91,72 @@
         return containsSubPlan("IgniteIndexScan(table=[[" + schema + ", " + tblName + "]], index=[" + idxName + ']');
     }
 
+    /**
+     * Ignite cost mather.
+     *
+     * @param cost TestCost to match.
+     * @return Matcher.
+     */
+    public static Matcher<String> containsCost(ServerStatisticsIntegrationTest.TestCost cost) {
```

Comment:
I don't think it's a good idea to have method like this. Otherwise we will be needed to fix all the tests uses this check when we start to tweak cost's weights.

Reply:
But only tests for cost weight should use it. We definitely should change such tests if we change weights.